### PR TITLE
Merge packages installation in v1alpha2 crs

### PR DESCRIPTION
**Pull request by fmuyassarov**
Merge packages installation into one line to reduce repetition.

### CODE REVIEW

File: crs/v1alpha2/controlplane_ubuntu.yaml [hunk 1]
```diff
@@ -42,22 +42,19 @@ spec:
...
-    - apt update -y
-    - apt install docker-ce docker-ce-cli containerd.io -y
-    - usermod -aG docker ubuntu
     - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
     - echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
-    - apt update
-    - apt install -y kubelet kubeadm kubectl
-    - systemctl enable --now kubelet
+    - apt update -y
+    - apt install |
```
Comment (maelk) [inline crs/v1alpha2/controlplane_ubuntu.yaml]: this is not correct syntax

File: crs/v1alpha2/workers_centos.yaml [hunk 2]
```diff
@@ -53,18 +53,15 @@ spec:
...
-        - yum install docker-ce-18.09.9 docker-ce-cli-18.09.9 containerd.io -y
-        - usermod -aG docker centos
-        - systemctl start docker
-        - systemctl enable docker
         - setenforce 0
         - sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
-        - yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
-        - systemctl enable --now kubelet
+        - yum install docker-ce-18.09.9 docker-ce-cli-18.09.9 containerd.io -y
+        - yum install kubelet kubeadm kubectl --disableexcludes=kubernetes -y
```
Comment (maelk) [inline crs/v1alpha2/workers_centos.yaml]: this can be merged

File: crs/v1alpha2/controlplane_ubuntu.yaml [hunk 3]
```diff
@@ -42,22 +42,19 @@ spec:
...
     - apt update -y
-    - apt install net-tools -y
-    - apt install -y gcc linux-headers-$(uname -r)
-    - apt install -y keepalived
-    - systemctl start keepalived
-    - systemctl enable keepalived
-    - apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common -y
+    - "apt install net-tools gcc linux-headers-$(uname -r) keepalived \
+          apt-transport-https ca-certificates curl gnupg-agent \
+          software-properties-common -y"
```
Comment (maelk) [inline crs/v1alpha2/controlplane_ubuntu.yaml]: maybe multiline yaml here ?

File: crs/v1alpha2/controlplane_centos.yaml [hunk 4]
```diff
@@ -38,22 +38,17 @@ spec:
...
-    - usermod -aG docker centos
-    - systemctl start docker
-    - systemctl enable docker
     - setenforce 0
     - sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
-    - yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
-    - systemctl enable --now kubelet
+    - "yum install gcc kernel-headers kernel-devel keepalived \
+          device-mapper-persistent-data lvm2 \
+          docker-ce-18.09.9 docker-ce-cli-18.09.9 containerd.io -y"
```
Comment (maelk) [inline crs/v1alpha2/controlplane_centos.yaml]: maybe multiline yaml here ?

File: crs/v1alpha2/controlplane_ubuntu.yaml (hunk 1 above)
Comment (fmuyassarov) [reply crs/v1alpha2/controlplane_ubuntu.yaml]: done

File: crs/v1alpha2/workers_centos.yaml (hunk 2 above)
Comment (fmuyassarov) [reply crs/v1alpha2/workers_centos.yaml]: done

File: crs/v1alpha2/controlplane_ubuntu.yaml (hunk 3 above)
Comment (fmuyassarov) [reply crs/v1alpha2/controlplane_ubuntu.yaml]: done

File: crs/v1alpha2/controlplane_centos.yaml (hunk 4 above)
Comment (fmuyassarov) [reply crs/v1alpha2/controlplane_centos.yaml]: done

File: crs/v1alpha2/workers_ubuntu.yaml [hunk 5]
```diff
@@ -57,17 +57,17 @@ spec:
...
-        - apt install docker-ce docker-ce-cli containerd.io -y
-        - usermod -aG docker ubuntu
         - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
         - echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
-        - apt update
-        - apt install -y kubelet kubeadm kubectl
-        - systemctl enable --now kubelet
+        - apt update -y
+        - apt install docker-ce docker-ce-cli containerd.io kubelet kubeadm kubectl -y
+        - systemctl enable --now docker kubelet
```
Comment (Xenwar) [inline crs/v1alpha2/workers_ubuntu.yaml]: you also need to include docker here.

Comment (maelk) [reply crs/v1alpha2/workers_ubuntu.yaml]: what do you mean ? docker is there, no ?

Comment (Xenwar) [reply crs/v1alpha2/workers_ubuntu.yaml]: I misread the removed part as current one. 
Please do ignore my comment.

**Pull request merged**